src/settings/handlers/DeviceSettingsHandler.ts

```diff
@@ -67,6 +68,13 @@ export default class DeviceSettingsHandler extends SettingsHandler {
             return val['value'];
         }
 
+        // Special case for old useIRCLayout setting
+        if (settingName === "layout") {
+            const settings = this.getSettings() || {};
+            if (settings["useIRCLayout"]) return Layout.IRC;
```

Comment:
Hmm, something seems a bit wrong here, since the old setting is an _account_ setting, but here we're reading from _device_ settings, right?

Reply:
The irc checkbox only ever set the value on the device level.

Reply:
Ah okay, makes sense, thanks!